server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleMapper.java

```diff
@@ -170,7 +170,7 @@ private static void setParams(Rules.Rule.Builder ruleResponse, RuleDto ruleDto,
   }
 
   private static void setCreatedAt(Rules.Rule.Builder ruleResponse, RuleDto ruleDto, Set<String> fieldsToReturn) {
-    if (shouldReturnField(fieldsToReturn, RuleNormalizer.RuleField.CREATED_AT) && ruleDto.getCreatedAt() != null) {
+    if (shouldReturnField(fieldsToReturn, RuleNormalizer.RuleField.CREATED_AT)) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "RuleNormalizer"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)